src/scripts/dungeons/Dungeon.ts

```diff
@@ -2634,13 +2634,23 @@ dungeonList['Granite Cave'] = new Dungeon('Granite Cave',
         ],
         legendary: [
             {loot: 'MediumRestore'},
-            {loot: 'Star Piece'},
+            {loot: 'Star Piece', ignoreDebuff : true},
             {loot: 'Hard Stone'},
         ],
         mythic: [{loot: 'Heart Scale'}],
     },
     410000,
-    [new DungeonBossPokemon('Mawile', 960000, 20), new DungeonBossPokemon('Nosepass', 660000, 20)],
+    [
+        new DungeonBossPokemon('Mawile', 960000, 20),
+        new DungeonBossPokemon('Nosepass', 660000, 20),
```

Comment:
Are these missing requirements?

Reply:
There are some more, where the old bosses are not hidden. Don't want to point all of them out, in case it's intended

Reply:
This was intended, to just add the trainer to the boss list but not totally replace the others.